docs/monitoring-usage.rst

```diff
@@ -0,0 +1,156 @@
+has a type, a display name, a description, and a list of
+:class:`~gcloud.monitoring.label.LabelDescriptor` instances.
+See the documentation about `Monitored Resources`_
+for more information.
+
+.. _Monitored Resources:
+    https://cloud.google.com/monitoring/api/v3/monitored-resources
+
+
+Metric Descriptors
+------------------
+
+The available metric types are defined by *metric descriptors*.
+They include `platform metrics`_, `agent metrics`_, and `custom metrics`_.
+You can list all of these with the
+:meth:`~gcloud.monitoring.client.Client.list_metric_descriptors` method::
+
+    >>> for descriptor in client.list_metric_descriptors():
+    ...     print descriptor.type
+
+See :class:`~gcloud.monitoring.metric.MetricDescriptor` and the
+`Metric Descriptors`_ API documentation for more information.
+
+.. _platform metrics: https://cloud.google.com/monitoring/api/metrics
+.. _agent metrics: https://cloud.google.com/monitoring/agent/
+.. _custom metrics: https://cloud.google.com/monitoring/custom-metrics/
+.. _Metric Descriptors:
+    https://cloud.google.com/monitoring/api/ref_v3/rest/v3/\
+    projects.metricDescriptors
+
+
+Time Series Queries
+-------------------
+
+A time series includes a collection of data points and a set of
+resource and metric label values.
+See :class:`~gcloud.monitoring.timeseries.TimeSeries` and the
+`Time Series`_ API documentation for more information.
+
+While you can obtain time series objects by iterating over a
+:class:`~gcloud.monitoring.query.Query` object, usually it is
+more useful to retrieve time series data in the form of a `pandas`_
+:class:`~pandas.DataFrame`, where each column corresponds to a
+single time series. For this, you must have ``pandas`` installed;
+it is not a required dependency of ``gcloud-python``.
+
+You can display CPU utilization across your GCE instances during
+the last five minutes as follows::
+
+    >>> METRIC = 'compute.googleapis.com/instance/cpu/utilization'
+    >>> query = client.query(METRIC, minutes=5)
+    >>> print query.as_dataframe()
+
+:class:`~gcloud.monitoring.query.Query` objects provide a variety of
+methods for refining the query. You can request temporal alignment
+and cross-series reduction, and you can filter by label values.
+See the client :meth:`~gcloud.monitoring.client.Client.query` method
+and the :class:`~gcloud.monitoring.query.Query` class for more
+information.
+
+For example, you can display CPU utilization during the last hour
+across GCE instances with names beginning with ``"mycluster-"``,
+averaged over five-minute intervals and aggregated per zone, as
+follows::
+
+    >>> from gcloud.monitoring import Aligner, Reducer
+    >>> METRIC = 'compute.googleapis.com/instance/cpu/utilization'
+    >>> query = client.query(METRIC, hours=1) \
+    ...               .select_metrics(instance_name_prefix='mycluster-') \
+    ...               .align(Aligner.ALIGN_MEAN, minutes=5) \
+    ...               .reduce(Reducer.REDUCE_MEAN, 'resource.zone')
```

Comment:
@jonparrott We weren't planning to throw this code over a wall for the core team to maintain.

@dhermes I changed this example in the documentation to use parentheses instead of backslashes. I put the dots after the line breaks so they line up.